### PR TITLE
Update DEPLOYMENT.md

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:

- Update DEPLOYMENT.md for production

**How to prepare for test**:
- Nothing to test

### Review:
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Chanjo if following the [GitHub flow][gh-flow] branching model which means that
 1. Deploy in productions by running the following commands:
     - `down`
     - `up`
-    - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-chanjo-prod.sh`
+    - `bbash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_chanjo -t chanjo -b master`
```
Comment (dnil) [inline DEPLOYMENT.md:24]:
```suggestion
    - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_chanjo -t chanjo -b master`
```
😊

**Review by dnil — APPROVED** [review]
Super - thank you! 🥇

File: DEPLOYMENT.md (hunk 1 above)
Comment (henrikstranneheim) [reply DEPLOYMENT.md:24]: Good catch 👍

**Comment by henrikstranneheim** [comment]
Deployed on Hasta (to test the script - there are no code changes):

```
θ62° [henrik.stranneheim@hasta:~] [P_base] 5s $ bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_chanjo -t chanjo -b master
Collecting git+https://github.com/Clinical-Genomics/chanjo@master
  Running command git clone -q https://github.com/Clinical-Genomics/chanjo /home/henrik.stranneheim/tmp/pip-req-build-g4fnplxk
  Cloning https://github.com/Clinical-Genomics/chanjo (to revision master) to ./tmp/pip-req-build-g4fnplxk
  Resolved https://github.com/Clinical-Genomics/chanjo to commit 13f80c592ade1693590992bc66af31b8c0600210
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: chanjo
  Building wheel for chanjo (setup.py): started
  Building wheel for chanjo (setup.py): finished with status 'done'
  Created wheel for chanjo: filename=chanjo-4.6.1-py2.py3-none-any.whl size=45077 sha256=2da8e336a737f522722923dee80e4d7003a4dadcf956106c2ffb9a256a6a7d9c
  Stored in directory: /home/henrik.stranneheim/tmp/pip-ephem-wheel-cache-wgpkn2yz/wheels/bb/47/20/c4fcc5c95c496a46fecd5ca5f778dc6f02885be44542b3fc76
Successfully built chanjo
Installing collected packages: chanjo
  Attempting uninstall: chanjo
    Found existing installation: chanjo 4.6.1
    Uninstalling chanjo-4.6.1:
      Successfully uninstalled chanjo-4.6.1
Successfully installed chanjo-4.6.1
Collecting git+https://github.com/Clinical-Genomics/chanjo@master
  Running command git clone -q https://github.com/Clinical-Genomics/chanjo /home/henrik.stranneheim/tmp/pip-req-build-j55ytmdd
  Cloning https://github.com/Clinical-Genomics/chanjo (to revision master) to ./tmp/pip-req-build-j55ytmdd
  Resolved https://github.com/Clinical-Genomics/chanjo to commit 13f80c592ade1693590992bc66af31b8c0600210
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: coloredlogs in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (15.0.1)
Requirement already satisfied: alchy in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (2.2.2)
Requirement already satisfied: click in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (8.0.4)
Requirement already satisfied: path.py in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (12.5.0)
Requirement already satisfied: toolz in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (0.12.0)
Requirement already satisfied: ruamel.yaml in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (0.17.21)
Requirement already satisfied: importlib_metadata in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (4.8.3)
Requirement already satisfied: pymysql in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (1.0.2)
Requirement already satisfied: sqlalchemy==1.3.* in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from chanjo==4.6.1) (1.3.24)
Requirement already satisfied: pydash>=3.4.7 in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from alchy->chanjo==4.6.1) (5.1.1)
Requirement already satisfied: humanfriendly>=9.1 in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from coloredlogs->chanjo==4.6.1) (10.0)
Requirement already satisfied: zipp>=0.5 in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from importlib_metadata->chanjo==4.6.1) (3.6.0)
Requirement already satisfied: typing-extensions>=3.6.4 in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from importlib_metadata->chanjo==4.6.1) (4.1.1)
Requirement already satisfied: path in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from path.py->chanjo==4.6.1) (16.2.0)
Requirement already satisfied: ruamel.yaml.clib>=0.2.6 in /home/proj/production/bin/miniconda3/envs/P_chanjo/lib/python3.6/site-packages (from ruamel.yaml->chanjo==4.6.1) (0.2.7)
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/henrik.stranneheim
done.
Log deploy... done.
chanjo, version 4.6.1
```

**Pull request merged**